### ISSUE TITLE
WCAG Accessibility Improvements - Low Hanging Fruits

### ISSUE THREAD

**Issue by fkellner** [geosolutions-it/MapStore2#10158]
## Description

![image](https://github.com/geosolutions-it/MapStore2/assets/28141677/3617e53e-f964-449e-8dc4-207ac0b121bb)

Running the MapStore dashboard through Lighthouse currently yields a bad accessibility score of 62, and the
map view does not score much better (66). As a german company using MapStore, we are required by law to 
put some effort into the accessibility of public facing websites.

Some changes that we did to improve the score are relatively painless and trivial, others, especially relating
to the labelling of buttons, introduce some small overhead. So, I will provide two PRs with our changes:

- One with the tiny changes that should not be an issue with the rest of the code (like setting document language, alt tag for logo), but already improve the score considerably
- One with the more complicated changes (mainly button labels with i18n). 

I hope the first one should not be a problem for you to merge, but the second one I am interested in your feedback, but it is not a high priority enhancement. 

A small accessibility issue in the navbar is currently caused by a `react-bootstrap`-Component, once you move on to the next major version, it should be fixed. 

**What kind of improvement you want to add?** (check one with "x", remove the others)
- [x] Minor changes to existing features
